Applications/Spire/Include/Spire/KeyBindings/AddRegionForm.hpp

```diff
@@ -0,0 +1,51 @@
+#ifndef SPIRE_ADD_REGION_FORM_HPP
+#define SPIRE_ADD_REGION_FORM_HPP
+#include <QWidget>
+#include "Spire/KeyBindings/KeyBindings.hpp"
+#include "Spire/Spire/ListModel.hpp"
+#include "Spire/Ui/RegionDropDownBox.hpp"
+#include "Spire/Ui/Ui.hpp"
+
+namespace Spire {
+
+  /** Displays a panel to add a region. */
+  class AddRegionForm : public QWidget {
+    public:
+    
+      /**
+       * Signals the submission of the region.
+       * @param submission The region to be submitted.
+       */
+      using SubmitSignal = Signal<void (const Nexus::Region& submission)>;
+
+       /**
+       * Constructs an AddRegionForm.
+       * @param regions A list of regions.
```

Comment:
Fix documentation, the documentation should make clear what the purpose of the parameter is.